src/components/avatar-builder/EmojiSelector.tsx

```diff
@@ -0,0 +1,360 @@
+import React, {
+  ComponentType,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { Dimensions, StyleSheet, View, ViewStyle } from 'react-native';
+import {
+  State as GestureHandlerState,
+  ScrollView,
+  TapGestureHandler,
+  TapGestureHandlerStateChangeEvent,
+} from 'react-native-gesture-handler';
+import { useSharedValue } from 'react-native-reanimated';
+import {
+  DataProvider,
+  LayoutProvider,
+  ProgressiveListView,
+} from 'recyclerlistview';
+import StickyContainer from 'recyclerlistview/dist/reactnative/core/StickyContainer';
+import 'string.fromcodepoint';
+import { ScrollEvent } from 'recyclerlistview/dist/reactnative/core/scrollcomponent/BaseScrollView';
+import { deviceUtils } from '../../utils';
+import { Categories } from './Categories';
+import EmojiContent from './EmojiContent';
+import EmojisListHeader from './EmojisListHeader';
+import EmojisLoader from './EmojisLoader';
+import EmojisStickyListItem from './EmojisStickyListItem';
+import InitialEmojis from './InitialEmojis';
+import TabsWithShadows from './TabsWithShadows';
+import { charFromEmojiObject } from './helpers/charFromEmojiObject';
+import getFormattedAllEmojiList, {
+  AllEmojiContentEntry,
+  AllEmojiEntry,
+  AllEmojiHeaderEntry,
+} from './helpers/getFormattedAllEmojiList';
+import { EmojiCategory, EmojiEntry } from './types';
+import { useTheme } from '@rainbow-me/theme';
+
+// TODO width attribute is temporary solution that will be removed as soon as I figure out why proper scaling does not work – comment from 2019
```

Comment:
I love dead TODOs xD

Reply:
I think I'm going to remove it, it is just misleading, we won't change it in the near future anyway